tests/features/joinup_subscription/unsubscribe.feature

```diff
@@ -77,6 +79,6 @@ Feature: Unsubscribe from collections
     And I should not see the link "Unsubscribe from all"
 
     And I should have the following collection content subscriptions:
-      | Southpark elementary | |
-      | Kenny's house        | |
-      | Koon's hideout       | |
+      | Southpark elementary |  |
+      | Kenny's house        |  |
+      | Koon's hideout       |  |
```

Comment:
:thinking:

Reply:
Wasn't me! It was the Ctrl+Alt+L that autoformatted it.